src/app/portlets/dot-edit-page/content/components/dot-edit-page-workflows-actions/dot-edit-page-workflows-actions.component.ts

```diff
@@ -10,16 +10,18 @@ import { Observable } from 'rxjs/Observable';
     templateUrl: './dot-edit-page-workflows-actions.component.html',
     styleUrls: ['./dot-edit-page-workflows-actions.component.scss']
 })
-export class DotEditPageWorkflowsActionsComponent implements OnInit {
-    @Input() inode: string;
+export class DotEditPageWorkflowsActionsComponent implements OnChanges {
+    @Input() pageState: any;
```

Comment:
It should be DotPage not any, we don't should use 'any' almost never

Reply:
done